client/capability-token-2way/capability-token.5.x.cs

```diff
@@ -0,0 +1,26 @@
+// In Package Manager, run:
+// Install-Package Twilio.Client -Pre
```

Comment:
No @joliveros, those instructions are for the [Package Manager console](https://docs.nuget.org/ndocs/tools/package-manager-console) which is widely used in VS as well.